docs/user-guide/tasks/ready.md

```diff
@@ -10,7 +10,8 @@ Check if a Kubernetes object exists and is ready.
 
 In the following example, the `ready` task checks if a deployment named `httpbin-prod` exists and its [availability condition](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) is set to true, and a service named `httpbin` exists.
 ```shell
-iter8 k launch \
+helm upgrade --install \
+--repo https://iter8-tools.github.io/iter8 --version 0.16 routeguide-test iter8 \
```

Comment:
Should this have a different name than `routeguide-test`? Maybe `ready-example`?

Reply:
Or possibly `httpbin-test`? Because the ready service is named such.